src/Serilog/LoggerConfigurationFullNetFxExtensions.cs

```diff
@@ -45,169 +39,11 @@ namespace Serilog
     /// </summary>
     public static class LoggerConfigurationFullNetFxExtensions
     {
-        const string DefaultOutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
+        //TODO: Need to confirm this is the best location for this default.  Used in File, Trace, RollingFile.  Do we move this to each sink?
+        public const string DefaultOutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
+        public const long DefaultFileSizeLimitBytes = 1L * 1024 * 1024 * 1024;
         const string DefaultConsoleOutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss} [{Level}] {Message}{NewLine}{Exception}";
```

Comment:
I think all of these should move to the respective `*LoggerConfigurationExtensions` classes. Doesn't matter if there is a bit of duplication IMHO, if it avoids dependencies.